packages/evmUtils/src/dataTypes/EvmChain/EvmChain.ts

```diff
@@ -12,65 +12,161 @@ import {
 } from '@moralisweb3/core';
 import { EvmUtilsConfig } from '../../config/EvmUtilsConfig';
 
+/**
+ * This can be any valid EVM chain in decimal or hex.
+ * @example "3"
```

Comment:
```suggestion
 * @example 3
```
Should not be a string